README.md

```diff
@@ -20,6 +18,33 @@
 [Click here to see the project]()
 
 
+## Getting Started
+
+**To get a local copy up and running follow these simple example steps.**
+## Setup
+
+- Clone this repository. 
+- Run ``npm i`` or ``yarn`` to install dependencies required to start the project
+- Make sure to use node v15.11.0 (npm v7.6.0) 
+- Run ``npm start``
+
+## Run Tests
+
+```
+npm run lint
+```
+
+## Author:
```

Comment:
[ OPTIONAL] As requested by the former code reviewer, please remove repeated sections in your readme file.